FRINXio/frinx-inventory-server
# FD-621 device inventory fixes

HOW TO TEST:

install/uninstall should work against uniconfig 6.0.0 (.39) and button should show current state (check:installed). i can provide you with actual physical device mount parameters for testing purpose.
